docs/feature_style.css

```
@@ -0,0 +1,156 @@
+    margin-bottom: 16px;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    color: var(--gold);
+
+    border: 2px solid rgba(200, 162, 77, 0.2);
+    border-radius: 16px;
+    background: rgba(200, 162, 77, 0.08);
+    transition: all 0.3sec ease;
```

Stylelint (17.11.1)

[error] Unknown unit "sec" (unit-no-unknown)

(unit-no-unknown)